static_frame/core/index_hierarchy.py

```diff
@@ -1337,7 +1347,7 @@ def size(self: IH) -> int:
         return self._blocks.size
 
     @property
-    def nbytes(self: IH) -> int:
+    def nbytes(self) -> int:
```

Comment:
I guess that type-checkers always correctly infer the type of the first argument when named `self`? I would have expected it be "better" to do `def nbytes(self: tp.Self) -> int`, but perhaps that is not the case!

Reply:
Right, you never need to do `self: tp.Self`!